src/test/fuzz/rpc.cpp

```diff
@@ -348,7 +348,7 @@ FUZZ_TARGET_INIT(rpc, initialize_rpc)
         return;
     }
     std::vector<std::string> arguments;
-    while (fuzzed_data_provider.ConsumeBool()) {
+    LIMITED_WHILE(fuzzed_data_provider.ConsumeBool(), 10000) {
```

Comment:
This seems a bit high? `ConsumeRPCArgument` might call `ConsumeArrayRPCArgument`, so the limit is 100'000'000. A user shouldn't be passing in more than 100 items. So any value between 100-1000 seems more appropriate.

See also https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=40648

Which is running base58 encoding (expensive) of one rpc arg 6000 times, thus times out. Your limit of 10000 wouldn't affect that, I think.

Reply:
ACK. I reduced it to 100.

I worry that reducing this will meaningfully affect test coverage since we presumably want to test what happens when you give gazillions of RPC arguments ... but probably a general-purpose fuzz test is the wrong place to try to hit that.

Reply:
Sorry, I also meant in the line above in this file. 100 * 100 is still 10k.